_partials/_migrate_set_up_source_and_target.md

```diff
@@ -5,7 +5,7 @@ databases are referred to as `$SOURCE` and `$TARGET` throughout this guide.
 This can be set in your shell, for example:
 
 ```bash
-export SOURCE=postgres://<user>:<password>@<source host>:<source port>/<dbname>
-export TARGET=postgres://<user>:<password>@<target host>:<target port>/<dbname>
+export SOURCE=postgres://<user>:<password>@<source host>:<source port>/<db_name>
```

Comment:
I changed this, because it's places before the `pg_dumpall` instructions that have the `-l $DB_NAME`. That way it's easier to make the connection when reading the instructions.